src/WindowManager.vala

```diff
@@ -721,17 +721,6 @@ namespace Gala {
                 out x, out y);
         }
 
-        public void dim_window (Meta.Window window, bool dim) {
```

Comment:
I think it'd be a good idea to keep this method around and move the logic to dim/"un-dim" an actor here.